### DISCUSSION TITLE
Issues with running the cats example

### DISCUSSION THREAD

**Discussion by ijohatsu** [miguelaeh/pipeless#160] [General]
Hi, 

I have installed Anaconda 2024.06-1 on my Ubuntu 24.04 LTS machine. Python version is 3.12.4

I have built pipeless from the source following the instructions in the documentation. It installed fine. However, when I am trying to run the cats example I am getting the following error: 

<pre>2024-10-24T05:19:34Z <font color="#26A269">INFO </font> pipeless_ai::stages::parser<font color="#5E5C64">]</font> ⚙️  Loading stages from .
<font color="#5E5C64">[</font>2024-10-24T05:19:34Z <font color="#26A269">INFO </font> pipeless_ai::stages::parser<font color="#5E5C64">]</font> ⏳ Loading stage &apos;cats&apos; from ./cats
<font color="#5E5C64">[</font>2024-10-24T05:19:34Z <font color="#26A269">INFO </font> pipeless_ai::stages::parser<font color="#5E5C64">]</font> 	Loading hook from ./cats/init.py
thread &apos;main&apos; panicked at src/stages/languages/python.rs:217:15:
Unable to create Python module from init hook: PyErr { type: &lt;class &apos;ModuleNotFoundError&apos;&gt;, value: ModuleNotFoundError(&quot;No module named &apos;cv2&apos;&quot;), traceback: Some(&lt;traceback object at 0x7914e4906540&gt;) }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
</pre>
 
Note, I have already installed opencv-python in the current conda environment and checked that opencv-python is indeed present as a site package. 

Would appreciate any help to resolve this problem. Thanks in advance.

**Comment by miguelaeh** [comment]
From the error it looks like you have to install open-cv: `"No module named 'cv2'"`
